### ISSUE TITLE
data loss after importing

### ISSUE THREAD

**Issue by alxgnon** [WuTheFWasThat/vimflowy#153]
When this app is installed as a Chrome extension, I get the following defective behaviour:

If I import a Vimflowy JSON file and close the window, I lose all changes to my document after the import.

## Steps to reproduce
1. Start with blank document
2. Go Settings > Import
3. Choose an exported Vimflowy JSON file
4. Click "Import!"
5. Wait for "Imported!" message
6. Close window

If I reopen Vimflowy after those steps, the document will be blank like on step 1.

**Comment by WuTheFWasThat** [comment]
chrome extension version doesn't save immediately.  this should definitely be either documented, and the UI should be improved.  See https://github.com/WuTheFWasThat/vimflowy/blob/master/assets/js/index.coffee#L168

try waiting for the log message of 'Saved', in the javascript console.  if it still is not persisting let me know and I'll try to reproduce.

**Comment by WuTheFWasThat** [comment]
actually, I just realized - maybe you have to be logged into chrome for chrome storage to work properly.  could that be the issue?

**Comment by alxgnon** [comment]
I am logged in the chrome and I'm sure it doesn't make a difference. Also, I was waiting enough time for the 'Saved' message, but thank you for letting me know about it.

I found the actual problem. Here is the error that happens right after the 'Saved' message:
```
Unchecked runtime.lastError while running storage.set: QUOTA_BYTES_PER_ITEM quota exceeded
    at chrome-extension://kjpjlojoecfhddakdfcddpafmehihmlp/js/index.js:223:36
```

I believe that means that my tree is too big for chrome storage.

I don't depend on this issue being fixed soon, because now the local web version works, so I am using that. It doesn't seem to have any problem with my large tree that I imported from WorkFlowy.
